tests/product/test_connectors.py

```diff
@@ -61,6 +61,96 @@ def test_basic_connector_add_remove(self):
             self.assert_has_default_connector(host)
         self._assert_connectors_loaded([['system'], ['tpch']])
 
+    def test_basic_connector_add_remove_coord_using_dash_h(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
+        self.run_prestoadmin('server start')
+        for host in self.cluster.all_hosts():
+            self.assert_has_default_connector(host)
+
+        self._assert_connectors_loaded([['system'], ['tpch']])
+
+        self.run_prestoadmin('connector remove tpch -H %(master)s')
+        self.run_prestoadmin('server restart')
+        self.assert_path_removed(self.cluster.master,
```

Comment:
Yes, I believe that in the long run, we should think about whether -H makes sense for each of the commands, and if it doesn't, we should error out. I will add tests for now anyways.